src/components/riveMonster/index.ts

```diff
@@ -0,0 +1,51 @@
+import { Rive, Layout, Fit, Alignment } from '@rive-app/canvas';
```

Comment:
Let's rename this to `rive-monster-component.ts`
Makes the file easier to find.

At a later time, let's decide on naming standards. It might also be covered in google typescript style guide.